web/saml.go

```diff
@@ -95,7 +95,7 @@ func completeSaml(c *Context, w http.ResponseWriter, r *http.Request) {
 		}
 		return
 	} else {
```

Comment:
While we're in here, should we just drop the `else` condition?

Reply:
Indeed. Good catch :)